src/v2/schema/message.js

```diff
@@ -0,0 +1,23 @@
+import { gql } from 'apollo-server-express';
+import logger from '../lib/logger';
+export const typeDef = gql`
+type Message {
+  id: String
+  kind: String
+  description: String
+}
+`;
+export const resolver = {
+  Query: {
+    messages: async (parent, args, { searchModel, req }) => {
+        const messages = []
+        const disabledClusters = await searchModel.resolveSearchCount({filters:[{ property: 'kind', values: ['cluster']},{ property: 'addon', values: ['search-collector=false']}, {property: 'name', values: ['!local-cluster']}]})
+        if (disabledClusters > 0 ){
+            logger.warn('Currently, search is disabled on some of your managed clusters. Some data might be missing from the console view. See _____ to enable search.');
```

Comment:
You need to add the message to the `messages` array.  You should see the message in the API response.